<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>f_LCD</class>
 <widget class="QWidget" name="f_LCD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Lb_ImgLCD">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>35</y>
     <width>102</width>
     <height>69</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LbNomLCD">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>10</y>
     <width>290</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>40</height>
    </size>
   </property>
   <property name="palette">
    <palette>
     <active/>
     <inactive/>
     <disabled/>
    </palette>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnRAZ">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>185</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Remise à Zéro</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnEnvoyer">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Envoyer</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="SpinBoxColonne">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>31</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="SpinBoxLigne">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>31</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="CheckBoxPrecis">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>95</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Précis</string>
   </property>
  </widget>
  <widget class="QLabel" name="LbColonne">
   <property name="geometry">
    <rect>
     <x>122</x>
     <y>110</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Colonnes</string>
   </property>
  </widget>
  <widget class="QLabel" name="LbLigne">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Lignes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnValeurPrecise">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Positionner</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="pTxETexte">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>110</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximumBlockCount">
    <number>2</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
